src/app/auth/components/log-in/log-in.component.ts

```diff
@@ -7,9 +9,25 @@ import { Component, OnInit } from '@angular/core';
 })
 export class LogInComponent implements OnInit {
 
-  constructor() { }
+  form!: FormGroup;
+  private authKey: string = 'Video_course_token';
+
+  constructor(private auth: AuthService) { }
 
   ngOnInit(): void {
+    this.buildForm()
+  }
+
+  onSubmit(): void {
+    this.auth.login(this.authKey, this.form.value.email)
+    console.log('Logged in successfully!')
+  }
+
+    private buildForm(): void {
+      this.form = new FormGroup({
```

Comment:
Wrong indentation. You can use autoformat on save setting in your IDE + prettier to prevent this.